doc/installation.md

```diff
@@ -8,31 +8,21 @@ issue is welcome. :)
 ## Preparation
 
 1. Install [Docker Community Edition](https://docs.docker.com/install/) on your Macbook.
-1. Build and run a dockerized MySQL server (In this example, called sqlflowdata) following [example/datasets](/example/datasets). Note that there is no need to install a local MySQL server, in which case you will have a port conflict in 3306. 
-1. Pull the latest SQLFlow Docker image: 
+1. Pull the latest SQLFlow "all-in-one" Docker image, which contains pre-built SQLFlow
+   binary, sample datasets (under `example/datasets`), and jupyter notebook.
 
    ```
-   docker pull sqlflow/sqlflow:latest
+   docker pull sqlflow/quickstart:latest
    ```
 
-## Running your first SQLFlow query
+## Running Your First SQLFlow Query
 
-After data is popularized in MySQL, let's test the installation from running a query in Jupyter Notebook:
-
-1. Start a Docker container that runs SQLFlow server and Jupyter Notebook. If you are
-   using Docker for Linux, please change `host.docker.internal:3306` to `localhost:3306`.
+1. Simply type the below command to start the service:
 
    ```
-   docker run --rm -it -p 8888:8888 sqlflow/sqlflow:latest \
-   bash -c "sqlflowserver --datasource='mysql://root:root@tcp(host.docker.internal:3306)/?maxAllowedPacket=0' &
-   SQLFLOW_SERVER=localhost:50051 jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root"
+   docker run --rm -p 8888:8888 sqlflow/quickstart:latest
```

Comment:
Should we leave option -RM up to user? In some cases, developers want to re-run docker

Reply:
good point, or shall we run the container in the background, adding `-d` so that the service will survive long?

Reply:
I personally agree, but removing -rm should also sync up other folks. I personally want to play around for a while but forgot we using -rm when it starts. 

We should probably simplified all the possible options for each commands and leave most of decisions to users with sufficient notes.

Reply:
Seems `docker start` to restart the container will run the script again when first-time `docker run` ran: https://stackoverflow.com/questions/26032929/does-docker-start-execute-the-cmd-command. So removing `--rm` should be fine.

Adding `-d` will cause user to have to run `docker logs` to get the notebook token, so will remove `--rm` in this PR only.

Reply:
Sounds good.